src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let ACTION_REQUIRED = '';
+  const stateCopy = { ...state };
+  const ARRAY = [Object.assign({}, stateCopy)];
```

Comment:
Variable names should use camelCase. ARRAY should be array.

Reply:
Fixed.